pom.xml

```diff
@@ -13,8 +13,8 @@
     <description>Allows JUnit-format test results to be published.</description>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/JUnit+Plugin</url>
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>2.7.3</jenkins.version>
```

Comment:
Is there something here that requires 2.7.3?   Might be worth holding it to 1.651 if we can since that has a pretty large install base according to stats.jenkins.io 

I know, I know, it's over the year support maximum by a month or two, but I figure there's no hard in taking an slightly older core dep if we don't need the newer things.

Reply:
Not really. In my company for example we do not use Blue Ocean because we use scripted pipeline and we have not plan conversion.
Our project has flow with two parallel stages that run integration tests on two different application server. Our needs is differentiate tests to understand from which step comes to understand on which WAS the test is failed.
Now we do know unless to archive surefire reports (xml) and check manually.

Reply:
Stats have shifted enough since July that I think we're good to bump to 2.7.3 without issue.  There's still some legacy users but I'm less concerned about them getting new features added to this plugin after that.

Reply:
> Not really. In my company for example we do not use Blue Ocean because we use scripted pipeline and we have not plan conversion.

@nfalco79 Sorry, but I don't see the relationship, FWIW. You can (and we definitely do) use Blue Ocean and it will render *any* Pipeline, be it Scripted or Declarative.
Sorry to digress, but I thought I wouldn't leave that statement without an answer for possibly passing-by people.

Reply:
@batmat that i remember blue ocean can render scripted pipeline only if non "standard" steps are written into a sort of step wrapper. And if i remember rigth also statements like if/def variable or function/for cycles and other groovy commands cause error in the parser when I had open the page of the our pipeline. So I had remove blue ocean and 15 related plugin after 3 hours.

Reply:
Well, not denying you could have hit some bug. If you're able to report it, that would be great. But yes, we use Blue Ocean in many many places, definitely too using Scripted pipelines, so this does work normally. Not being the case looks like this would be great to be reported somewhere.